lib/Doctrine/Migrations/Configuration/Configuration.php

```diff
@@ -42,6 +42,9 @@ class Configuration
     /** @var int */
     private $migrationsColumnLength = 255;
 
+    /** @var string */
+    private $migrationsExecutedAtColumnName = 'executedAt';
```

Comment:
Should probably be `executed_at` to match table name and also avoid case sensitivity mess.